infrastructure/security.tf

```diff
@@ -25,6 +30,53 @@ resource "aws_security_group_rule" "data_refinery_worker_custom" {
   security_group_id = "${aws_security_group.data_refinery_worker.id}"
 }
 
+# Allow the Nomad HTTP API to be accessible by this security group. See:
+# https://www.nomadproject.io/guides/cluster/requirements.html#ports-used
+resource "aws_security_group_rule" "data_refinery_worker_nomad" {
+  type = "ingress"
+  from_port = 4646
+  to_port = 4646
+  protocol = "tcp"
+  self = true
+  security_group_id = "${aws_security_group.data_refinery_worker.id}"
+}
+
+# Allow Nomad RPC calls to go through to each other. See:
+# https://www.nomadproject.io/guides/cluster/requirements.html#ports-used
+resource "aws_security_group_rule" "data_refinery_worker_nomad_rpc" {
+  type = "ingress"
+  from_port = 4647
+  to_port = 4647
+  protocol = "tcp"
+  self = true
+  security_group_id = "${aws_security_group.data_refinery_worker.id}"
+}
+
+# Allow Nomad Servers to gossip with each other over TCP. See:
+# https://www.nomadproject.io/guides/cluster/requirements.html#ports-used
+resource "aws_security_group_rule" "data_refinery_worker_nomad_serf_tcp" {
+  type = "ingress"
+  from_port = 4648
+  to_port = 4648
+  protocol = "tcp"
+  self = true
+  security_group_id = "${aws_security_group.data_refinery_worker.id}"
+}
+
+# Allow Nomad Servers to gossip with each other over UDP. See:
+# https://www.nomadproject.io/guides/cluster/requirements.html#ports-used
+resource "aws_security_group_rule" "data_refinery_worker_nomad_serf_udp" {
+  type = "ingress"
+  from_port = 4648
+  to_port = 4648
+  protocol = "udp"
+  self = true
+  security_group_id = "${aws_security_group.data_refinery_worker.id}"
+}
+
+# Allow SSH connections to Nomad instances. This is pretty much the
+# only way to debug issues until we get the Nomad UI up and running.
+# THIS DEFINITELY NEEDS TO BE REMOVED LONG TERM!!!!!!!!!!
```

Comment:
Add `# XXX` so we can `ag` for it before prod deploy!

Reply:
:+1: